test/extension/utils.test.js

```diff
@@ -137,6 +138,10 @@ describe('Test extension utils', () => {
     const spy = sinon.spy(window.chrome.storage.sync, 'get');
     await utils.getConfig('sync', 'name');
     expect(spy.calledWith('name')).to.be.true;
+    const res = await new Promise((resolve) => {
+      utils.getConfig('sync', 'name', resolve);
```

Comment:
isn't `getConfig()` already async? why the callback variant?

Reply:
Good question... some of these functions already had a callback variant, so I added one to all for consistency. But you're right, actually they don't need both. I'll remove them.